### ISSUE TITLE
SavedSearchQuery: save relevant parameters only

### ISSUE THREAD

**Issue by lpirl** [sdm15stream2/serler#18]
Instead of saving the whole URL, it would probably be more flexible (think of URL changes) to store only the URL parameters that are relevant for the search.

We could probably pass the corresponding `params` in `SearchController`'s `index` to the view, populate a field in the form "save search query" so that we can store those parameters for instances of `SavedSearchQuery`.
(Also, this would probably imply to rename the attribute `address` of `SavedSearchQuery` to `search_parameters` or similar.)

**Comment by klauszhang** [comment]
you may use this one:
http://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-original_fullpath

**Comment by lpirl** [comment]
True, that would work (like it does now). I think we don't need to store the path, since we can always get it through URL helpers. That way, if the URL `/search` for example changes, the saved queries continues working.

**Comment by klauszhang** [comment]
Well you can't do this now because once the advance search kicked in there
will be a lot of changes. Easier to save just urls.

**Comment by lpirl** [comment]
I meant saving URL parameters - that should be save with the advanced search as well as long as it is based on GET parameters. I won't do it anyways (right now), I just wanted to document this flaw. :)

**Comment by klauszhang** [comment]
well. for me, if this store like an object. It sounds like another user story. I tried to store it as an key:value pair. but I don't see it make a lot of scene, I need to build a new url from that pair, and i also need to read information from search object.... so I stored it as a string to make everything just enough.
To explain why it is stored as a string.

However, this also have a lot of security issues. which can't be ignored and should be a part of non functional requirement.
